### ISSUE TITLE
A System Error Occurred: -2147221164

### ISSUE THREAD

**Issue by ghost** [seathasky/FF14-MAC_ModSupport#35]
I am trying to install FFXIV with the native (NA) launcher and receive this error:

![Screen Shot 2021-11-28 at 5 12 47 PM](https://user-images.githubusercontent.com/84574720/143734865-af9c2824-3764-48df-bd4d-ab90dc0fd2c2.png)

Log:
```
Running launcher: FINAL FANTASY XIV (StartMenu.C^3A_ProgramData_Microsoft_Windows_Start+Menu/Programs/SQUARE ENIX/FINAL FANTASY XIV ONLINE/FINAL FANTASY XIV.lnk)
Command-line options: 
Bottle: Final Fantasy XIV Online
Debug channels: 
Extra environment variables (null)

CrossOver 21.0.0.33720
macOS Version 10.16 (Build 21A559)
Install location: /Applications/CrossOver.app
Mac Locale: en_KR

CXConfig->read(/Applications/CrossOver.app/Contents/SharedSupport/CrossOver/etc/CrossOver.conf)
CXConfig->read(/Users/demo/Library/Application Support/CrossOver/CrossOver.conf)
69114: Grabbing the '/var/folders/1v/j6f1b0cs6s7c49nx08k70bym0000gn/T//.wine-501/bottle-100000f-1ec00a.lock' lock
69114: Got the '/var/folders/1v/j6f1b0cs6s7c49nx08k70bym0000gn/T//.wine-501/bottle-100000f-1ec00a.lock' lock
CXConfig->read(/Users/demo/Library/Application Support/CrossOver/Bottles/Final Fantasy XIV Online/cxbottle.conf)
69114: Releasing the '/var/folders/1v/j6f1b0cs6s7c49nx08k70bym0000gn/T//.wine-501/bottle-100000f-1ec00a.lock' lock
Bottle environment variables:
 CX_BOTTLE_CREATOR_APPID -> com.codeweavers.c4.7892
 WINEESYNC -> 1
 GL_SHADER_DISK_CACHE_SKIP_CLEANUP -> 1
 MVK_CONFIG_FAST_MATH_ENABLED -> 1
 HARMONY_DEBUG -> 1
 DXVK_HUD -> fps,compiler
 "DXVK_ASYNC" -> 1
CXRWConfig->new(/Users/demo/Library/Application Support/CrossOver/Bottles/Final Fantasy XIV Online/cxmenu.conf)
'/Users/demo/Library/Application Support/CrossOver/Bottles/Final Fantasy XIV Online/cxmenu.conf' not modified -> no need to save
Starting 'StartMenu.C^3A_ProgramData_Microsoft_Windows_Start+Menu/Programs/SQUARE ENIX/FINAL FANTASY XIV ONLINE/FINAL FANTASY XIV.lnk'
system encoding='UTF-8'
Exec-ing '"/Applications/CrossOver.app/Contents/SharedSupport/CrossOver/bin/wine" --bottle "Final Fantasy XIV Online" --check --wait-children --start "C:/ProgramData/Microsoft/Windows/Start Menu/Programs/SQUARE ENIX/FINAL FANTASY XIV ONLINE/FINAL FANTASY XIV.lnk" --cx-log -'
esync: up and running.
0094:err:plugplay:process_IOService_Device object 0x7703
0094:err:plugplay:process_IOService_Device object 0xaa03
0094:err:plugplay:process_IOService_Device object 0xab03
0094:err:plugplay:process_IOService_Device object 0xad03
0024:fixme:exec:SHELL_execute flags ignored: 0x00000100
0024:fixme:exec:SHELL_execute flags ignored: 0x00004100
00f0:fixme:ver:GetCurrentPackageId (0136FEF0 00000000): stub
0100:fixme:msg:ChangeWindowMessageFilter 4a 00000001
0104:fixme:ver:GetCurrentPackageId (01D4FEF0 00000000): stub
0100:fixme:ieframe:PersistStreamInit_InitNew (001EBE18)
0100:fixme:ieframe:ViewObject_SetAdvise aspects and/or flags not supported yet
0100:fixme:ieframe:ViewObject_Draw (001EBE18)->(1 -1 00000000 00000000 00000000 0002004F 001E8CEC 001E8CEC 00000000 00000000)
0100:fixme:ieframe:WebBrowser_put_RegisterAsDropTarget (001EBE18)->(0)
0100:fixme:urlmon:create_object Could not find object for MIME L"text/html"
0100:fixme:ieframe:handle_navigation_error Navigate to error page
0100:fixme:ieframe:bind_to_object BindToObject failed: 80040154
0100:fixme:urlmon:InternetBindInfo_GetBindString not supported string type 20
010c:fixme:secur32:schan_get_cipher_algid Don't know CALG for encryption algorithm 2, returning 0
010c:fixme:secur32:schan_imp_get_max_message_size Returning 1 << 14.
010c:fixme:secur32:schan_get_cipher_algid Don't know CALG for encryption algorithm 2, returning 0
010c:fixme:wininet:create_cache_entry INTERNET_FLAG_NEED_FILE is not supported correctly
0100:fixme:urlmon:create_object Could not find object for MIME L"text/html"
0100:fixme:ieframe:handle_navigation_error Navigate to error page
0100:fixme:ieframe:ViewObject_SetAdvise aspects and/or flags not supported yet
0100:fixme:msg:ChangeWindowMessageFilter 4a 00000002
0080:err:rpc:I_RpcReceive we got fault packet with status 0x1c010003
```

MacOS 12.1, MacBook Air M1.  I tried to launch iexplore.exe and it does not render anything even though the address bar upgrades the request from http to https.  In the past, I have also seen this error:

`010c:fixme:secur32:schan_get_cipher_algid Don't know CALG for encryption algorithm 4, returning 0`

Is the gnutls that comes with crossover is outdated?  Not sure what else can cause this.

**Comment by seathasky** [comment]
We don't really offer support for the default launcher as it is extremely buggy in CX21.

**Comment by ghost** [comment]
This is fixed in the latest 21.1.0 update that CX pushed out!

**Comment by seathasky** [comment]
Good to know!
